tests/src/Kernel/Config/MigrationConfigOverridesTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Drupal\Tests\stanford_migrate\Kernel\Config;
+
+use Drupal\file\Entity\File;
+use Drupal\Tests\stanford_migrate\Kernel\StanfordMigrateKernelTestBase;
+
+/**
+ * Class MigrationConfigOverridesTest.
+ *
+ * @group stanford_migrate
+ * @coversDefaultClass \Drupal\stanford_migrate\Config\MigrationConfigOverrides
+ */
+class MigrationConfigOverridesTest extends StanfordMigrateKernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'test_stanford_migrate',
+    'stanford_migrate',
+    'migrate_plus',
+    'migrate',
+    'node',
+    'user',
+    'system',
+    'ultimate_cron',
+    'file'
+  ];
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+    $this->installEntitySchema('file');
+  }
+
+  /**
+   * After setting the state, the config overrides should set the path data.
+   */
+  public function testConfigOverrides() {
+    $source = \Drupal::configFactory()
+      ->get('migrate_plus.migration.stanford_migrate')
+      ->get('source');
+    $this->assertArrayNotHasKey('path', $source);
+
+    \Drupal::service('file_system')->copy($this->root . '/core/misc/druplicon.png', 'public://example.jpg');
```

Comment:
Are you testing with a jpg rather than a csv just for convenience, to make sure the path works?

Reply:
Yeah since we don't really want to test the import since every migration is different.